underlay/src/main/resources/config/datamapping/aouRT/entity/ingredientOccurrence/all.sql

```diff
@@ -14,19 +14,17 @@ SELECT
   de.visit_occurrence_id,
   vo.visit_concept_id,
   vc.concept_name AS visit_concept_name
-
 FROM `${omopDataset}.drug_exposure` AS de
-
-JOIN `${omopDataset}.person` AS p
-    ON p.person_id = de.person_id
-
+JOIN `${omopDataset}.person` AS p ON p.person_id = de.person_id
 JOIN `${omopDataset}.concept` AS ic
     ON ic.concept_id = de.drug_concept_id
+        AND de.drug_concept_id IS NOT null
+        AND de.drug_concept_id != 0
+        AND ic.domain_id = 'Drug'
+        AND (
+        (ic.vocabulary_id = 'ATC' AND ic.standard_concept = 'C')
+        OR (ic.vocabulary_id IN ('RxNorm', 'RxNorm Extension') AND ic.standard_concept = 'S')
+        )
```

Comment:
I would remove this check as we should be pulling all drug data except where drug_concept_id is not null and drug_concept_id not equal to 0